app/models/evss_claim_document.rb

```diff
@@ -48,7 +48,8 @@ class EVSSClaimDocument < Common::Base
     'L478' => 'Medical Treatment Records - Furnished by SSA',
     'L702' => 'Disability Benefits Questionnaire (DBQ)',
     'L703' => 'Goldmann Perimetry Chart/Field Of Vision Chart',
-    'L827' => 'VA Form 21-4142a - General Release for Medical Provider Information'
+    'L827' => 'VA Form 21-4142a - General Release for Medical Provider Information',
+    'L1489' => 'Automated Review Summary Document'
```

Comment:
This hash was updated in accordance with guidance from [#evss-dev](https://dsva.slack.com/archives/C1VBAHWQL/p1662661498171129?thread_ts=1662659771.674139&cid=C1VBAHWQL) that EVSS and VBMS do in fact support this type ID -- I am not just relying on wishful thinking 😄